include_core/AtomicSupport.hpp

```diff
@@ -259,12 +259,7 @@ class VM_AtomicSupport
 #elif defined(_MSC_VER)
 		_ReadWriteBarrier();
 #elif defined(__GNUC__)
-#if defined(J9X86) || defined(J9HAMMER)
-		asm volatile("":::"memory");
-		/* TODO investigate whether or not we should call this
-		asm volatile("sfence" ::: "memory");
-		*/
-#elif defined(ARM) /* defined(J9X86) || defined(J9HAMMER) */
+#if defined(ARM)
```

Comment:
line 268 needs an update `#endif /* defined(J9X86) || defined(J9HAMMER) */` to fix the comment